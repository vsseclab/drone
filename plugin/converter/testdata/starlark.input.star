def main(ctx):
  return {
    "kind": "pipeline",
    "name": "build",
    "steps": [
      {
        "name": ctx.input.stepName,
        "image": ctx.input.image,
        "commands": [
            ctx.input.commands
        ]
      }
    ]
  }
