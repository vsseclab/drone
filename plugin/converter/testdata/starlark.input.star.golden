{"kind": "pipeline", "name": "build", "steps": [{"name": "my_step", "image": "my_image", "commands": ["my_command"]}]}